kernel/posixtest.py

```diff
@@ -60,7 +60,14 @@ def setUp(self):
     def test(self):
 
         os.chdir(self.sourcedir)
-        process.system('./run_tests THR')
+        process.system('./run_tests THR', ignore_status=True)
+
+        logfile = os.path.join(self.logdir, "stdout")
+        failed_tests = process.system_output(
```

Comment:
It is better to do this through parsing the output via `process.system_output('./run_tests THR')` or native file handling of stdout.

Reply:
@narasimhan-v thanks for review , sure make sense will adopt the changes